observations/obs_converters/GSI2DART/work/mkmf_gsi_to_dart

```diff
@@ -4,24 +4,26 @@
 # by UCAR, "as is", without charge, subject to all terms of use at
 # http://www.image.ucar.edu/DAReS/DART/DART_download
 #
-# DART $Id$
-
+# usage: mkmf_gsi_to_dart 
+#
+# gsi_to_dart requires mpi libraries
+# it will be compiled with the mpi
+# libraries and can run with multiple cooperating processes.
+#
 # we need to preprocess kinds.F90.   the following works on gfortran
 # 
-
 # but if it doesn't, you can try this to explicitly preprocess F90 -> f90 first,
 # and change the path_names file to include the .f90 file instead of .F90:
 #
 # rm -f ../enkf/kinds.f90
 # /usr/bin/cpp -P -D_REAL8_ -traditional-cpp ../enkf/kinds.F90 > ../enkf/kinds.f90
 
-../../../../build_templates/mkmf -p gsi_to_dart -t ../../../../build_templates/mkmf.template   \
- -c " -D_REAL8_" -a "../../../.." -w path_names_gsi_to_dart  # Note the "-w" for MPI build
+echo "Making Makefile with MPI"
+touch using_mpi_for_gsi_to_dart
```

Comment:
This is needed because the presence of the file is what kicks off the use of the mpicommand.